tests/test_model.py

```diff
@@ -19,3 +23,27 @@ def test_querying(create, tmpdir):
     assert len(set(db.todos())) == 9
     assert len(set(db.todos(lists='ab'))) == 6
     assert len(set(db.todos(lists='ab', location='a'))) == 2
+
+
+def test_retain_tz(tmpdir, runner, create, default_database):
+    create(
+        'ar.ics',
+        'SUMMARY:blah.ar\n'
+        'DUE;VALUE=DATE-TIME;TZID=ART:20160102T000000\n'
```

Comment:
@geier Nope, I just used my own because I originally copied the line from some real todo.
Argentina is pretty disorganized when it comes to timezones; we have just one (UTC-3).

We don't have DST, *except* some years when a presidential order added a one-time DST, and provinces could individually adhere to it (or not). Since some provinces adhered to DST, we then had different timezones.

Since different provinces adhered to the order on different one-time DST-additions, this meant that they basically spawned their own timezone.

So, as far as IT does, we have potentially 23 timezones, which *are usually* the same, but will vary every several years.

To make matters worse, this all tends to be done a month or two before DST hits.